api-reference/10 UI Widgets/dxScheduler/1 Configuration/onAppointmentDblClick.md

```diff
@@ -18,7 +18,7 @@ The initial appointment.
 #include common-ref-elementparam with { element: "clicked" }
 
 ##### field(e.cancel): Boolean
-Allows you to cancel execution of the default appointment click handler.
+If **true**, prevents the appointment details form from opening on double click.
```

Comment:
I would try to avoid 'form from opening'. It is not a rule, but in my opinion in this particular case these words look a bit odd together. 

How about 'If **true**, prohibits the appointment details form opening on double click'?